packages/common/src/ui/forms/Modal/ModalErrorMessage.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Typography } from '@mui/material';
+import { useFormContext, get, FieldErrors, Message } from 'react-hook-form';
+
+export type ModalErrorMessageProps = {
+  errors?: FieldErrors;
+  name: string;
+  message?: Message;
+};
+export const ModalErrorMessage: React.FC<ModalErrorMessageProps> = ({
+  errors,
+  name,
+  message,
+}) => {
+  const methods = useFormContext();
```

Comment:
Think I'm missing a trick here.

What *I* think is happening, can you correct where I'm wrong:

You use `useFormState` in the parent, which I think is equal to `useFormContext().formState` and pass down `errors` which is equal to `useFormContext.formState.errors` and then do the long version and pass both values which are equal into `get`?